src/vpd_parser_main.cpp

```diff
@@ -64,8 +62,9 @@ int main(int argc, char** argv)
         // Below are two different ways of parsing the VPD.
         if (!configFilePath.empty())
         {
+            vpd::logging::logMessage("Config file path recieved" +
```

Comment:
Why this additional log here?

Reply:
Previously this log was displayed if config file option is used. As the macro check was removed the log is getting displayed in both the cases with and without config Json.
So moved the log to config file check

Reply:
We need this but why we need two log messages? Club both of them in one and come up with a new one.

Reply:
done